code/modules/atmospherics/FEA_airgroup.dm

```diff
@@ -225,10 +226,7 @@
 						if(connection_difference > 0)
 							self_border.consider_pressure_difference(connection_difference, get_dir(self_border,enemy_border))
 						else
-							var/turf/enemy_turf = enemy_border
-							if(!isturf(enemy_turf))
-								enemy_turf = enemy_border.loc
-							enemy_turf.consider_pressure_difference(-connection_difference, get_dir(enemy_turf,self_border))
+							enemy_border.consider_pressure_difference(-connection_difference, get_dir(enemy_border,self_border))
```

Comment:
the amount of indents on this line is causing it to show up weird in github. Looks correct tho.